Fix ALL typos across all .md files

nick-w-nick opened the pull request:
Hello, I went through each and every .md file within the /docs folder individually and used a spell checker to find and fix any typos/errors.

Across all 36 markdown files currently in the /docs folder, 18 of them needed one or more spelling corrections.
This PR does not also technically target grammatical errors, but it should naturally fix a handful of them once merged.

Let me know if there are any issues with the PR or if any changes need to be made.

Thanks!

Unitech commented:
Hi! Thanks for this, I've refactored the whole doc, could you run your script again and open a PR? Thank you

nick-w-nick commented:
> Hi! Thanks for this, I've refactored the whole doc, could you run your script again and open a PR? Thank you

@Unitech I have updated the PR with the spelling/grammatical fixes once again and it should now be up to date with the master branch.

Please review the changes and let me know if you need any additional modifications prior to merging.

Thanks!
